### PR TITLE
Image builder address design review comments

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Restyle the login section
- Fix the disable overlap
- Always display Got an app that isn’t in the Snap Store
- Conditionally display email in build page
- Update the footer content 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/core/build
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check I have addressed al lthe comment here: https://github.com/canonical-web-and-design/ubuntu.com/pull/6695#issuecomment-600053523

### CODE REVIEW

File: templates/core/build.html [hunk 1]
```diff
@@ -31,239 +31,226 @@ <h1>Your Ubuntu image is building</h1>
...
-          <input type="hidden" name="FullName" value="{{ openid.fullname }}" />
-          <input type="hidden" name="Email" value="{{ openid.email }}" />
-          <p><button class="p-button--positive js-build-button">Build image</button></p>
-        </form>
-      </div>
+<section class="p-strip--light is-shallow  {% if not openid %}u-disable{% endif %}">
+  <div class="row">
+    <div class="col-8">
+      <form method="POST" id="build-form">
+        <p>We’ll email you at {{ openid.email }} once the build is ready.</p>
```
Comment (solazio) [inline templates/core/build.html]: I don't think the text makes too much sense if there is no emali.

**Comment by anthonydillon** [comment]
Oh sorry you need the lp token to trigger builds but that part has been tested and hasn't changed in this branch.

File: templates/core/build.html (hunk 1 above)
Comment (anthonydillon) [reply templates/core/build.html]: When did you see it without a email?

Comment (matthewpaulthomas) [reply templates/core/build.html]: Yep, in #6695 I said “It’s a bit unpolished that when signed out, we say ‘We’ll email you at once the build is ready.’ Could we drop the ‘at’ in that case?’”

Comment (solazio) [reply templates/core/build.html]: @anthonydillon it's at the bottom of the page.

Comment (anthonydillon) [reply templates/core/build.html]: Ah, I see. I have hidden the message if there is no email.

**Pull request merged**